custom_components/smartlife/sensor.py

```diff
@@ -1046,6 +1046,56 @@ class SmartLifeSensorEntityDescription(SensorEntityDescription):
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    # Button or Multi-button device
+    # https://developer.tuya.com/en/docs/iot/f?id=Kbeoa30s4fcdf
+    "wxkg": (
+        SmartLifeSensorEntityDescription(
+            key=DPCode.SWITCH_MODE_1,
+            name="Click",
```

Comment:
You can take a look at the app display and suggest using Switch1

Reply:
I mean the name attribute.

Reply:
@wilwac I think this is a valid comment and it's the way the wkcz and kg categories have been coded in the switch.py file. So having unique names for the Switch sensors does make sense.